src/libs/actions/SharedItem.js

```diff
@@ -0,0 +1,23 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+
+/**
+ * Saves shared item to storage.
+ *
+ * @param {Object} sharedItem
+ */
+function set(sharedItem) {
+    Onyx.set(ONYXKEYS.SHARED_ITEM, sharedItem);
+}
+
+/**
+ * Clears shared item from storage.
+ */
+function clear() {
+    Onyx.set(ONYXKEYS.SHARED_ITEM, null);
```

Comment:
Can also use [`Onyx.remove()`](https://github.com/Expensify/react-native-onyx/blob/d8e255f0febe35e33e0f76ab6400d608689a0614/lib/Onyx.js#L341-L350). 

Though I'm not sure I would have gone with `Onyx` here. Seems easier to just store the shared item in memory.